### PR TITLE
[libvpx] Use vcpkg_configure_make on UNIX

### PR DESCRIPTION
**Pull request by Thomas1664**
**Describe the pull request**

- #### What does your PR fix?
  Build with make on Windows
  Use `vcpkg_configure_make()` and `vcpkg_install_make()` on UNIX
  Remove unofficial CMake config file because port has pc files

### CODE REVIEW

File: ports/libvpx/portfile.cmake [hunk 1]
```diff
@@ -14,34 +12,17 @@ vcpkg_from_github(
         0004-remove-library-suffixes.patch
 )
 
-vcpkg_find_acquire_program(PERL)
 
-get_filename_component(PERL_EXE_PATH ${PERL} DIRECTORY)
+if("realtime" IN_LIST FEATURES)
+    set(OPTIONS "${OPTIONS} --enable-realtime-only")
```
Comment (dg0yt) [inline ports/libvpx/portfile.cmake:17]: OPTIONS is a list, not a string, for `vcpkg_configure_make`.

File: ports/libvpx/portfile.cmake [hunk 2]
```diff
@@ -12,36 +10,20 @@ vcpkg_from_github(
         0002-Fix-nasm-debug-format-flag.patch
         0003-add-uwp-v142-and-v143-support.patch
         0004-remove-library-suffixes.patch
+        allow-unknown-options.patch
 )
 
-vcpkg_find_acquire_program(PERL)
-
-get_filename_component(PERL_EXE_PATH ${PERL} DIRECTORY)
+set(OPTIONS)
```
Comment (Neumann-A) [inline ports/libvpx/portfile.cmake:16]:
```suggestion
set(OPTIONS "")
```

**Review by FrankXie05 — DISMISSED** [review]

**Comment by Hoikas** [comment]
Removing the unofficial configs breaks any downstream users who might be using them (eg me). This increases the complexity of updating to new revisions of the main catalog, which isn't a good thing, IMO.

**Review by vicroms — CHANGES_REQUESTED** [review]
As @Hoikas pointed out we must keep unofficial targets around to avoid breaking users. 
We can make the unofficial targets into a static file that forwards to the official targets.

**Comment by Thomas1664** [comment]
Unsupported on arm64_android?

`ld: error: /tmp/vpx-conf-812-9505.o is incompatible with aarch64linux`

**Comment by dg0yt** [comment]
> Unsupported on arm64_android?
> 
> `ld: error: /tmp/vpx-conf-812-9505.o is incompatible with aarch64linux`

It is probably missing arguments as part of CC/CXX/LD. Needs #31228

**Pull request closed without merging**